.circleci/config.yml

```diff
@@ -171,7 +206,8 @@ workflows:
   build-test-deploy:
     jobs:
       - api-build-test-deploy
-      - ui-build-test-deploy
+      - api-local-tests
```

Comment:
Hmm, so technically it would be nice if we blocked deployment on all of the tests passing, but I'd probably take the tradeoff for shorter CI turnaround if I had to pick.

Reply:
Yeah, that was my thought.

If we have a policy of not merging until everything on your PR is green, then the checks here are just a safety measure anyway.